### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yilikal250**
This is the portfolio of a software-developer with HTML ,CSS, images and additional information.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="upper">
+      <div class="toolbar">
+        <p id="name">Yilikal Teshome</p>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly change this p tag to an a tag as this is your logo for the website
- [ ] Kindly remove white space before header check the screenshot below ⬇️

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <div>
+        <h1>Hey There. I'm Yilikal</h1>
+        <h2>I'm a Software Developer</h2>
+        <p class="detail">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="sidepic">
+          <img src="Illustration(1).png" alt="" />
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add this background image with css not img 👍🏿

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        </div>
+      </div>
+      <div>
+        <nav class="icons">
+          <img src="github.png" alt="" />
+          <img src="linked-in-logo-of-two-letters.png" alt="" />
+          <img src="peace.png" alt="" />
+          <img src="twitter(1).png" alt="" />
+          <img src="m.png" alt="" />
+        </nav>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly use the ul and li, a tags for the social media icons, it should look like this:
 `<li><a href="#">medium</a></li>`

- [ ] Please add alt text to all images this is important so when browsers are unable to render a page properly a text is displayed 👍🏿 


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
dd  | ggg

- [ ] Kindly remove background color from the social media icons

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      <div class="toolbar">
+        <p id="name">Yilikal Teshome</p>
+        <nav class="drop">
+          <img id="icondrop" src="dropbox.png" alt="" />
+        </nav>
+      </div>
+    </header>
+    <section class="lower">
+      <div>
+        <h1>Hey There. I'm Yilikal</h1>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add more the spacing between the hx and header tag, check screenshot below ⬇️ 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
dd  | ggg

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Yilikal250, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good design


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Always add a descriptive pull request message and title describing changes made in your pull request 🤝 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="upper">
+      <div class="toolbar">
+        <a id="name">John Doe</a>
+        <nav class="drop">
```
Comment (MeqdamAlqudah) [inline index.html:22]: - [ Optional] Please write your name instead of John Doe except your name is John Doe😅

**Review by MeqdamAlqudah — APPROVED** [review]
# Hi @Yilikal250 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes made by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**